pkg/migration/controllers/migration.go

```diff
@@ -631,12 +635,39 @@ func (m *MigrationController) objectToBeMigrated(
 			return false, nil
 		}
 		return true, nil
-	case "Secret":
```

Comment:
Why is secret object removed from migration?

Reply:
It's not removed. This function just returns whether an object should be migrated. Previously we were not returning secrets for secret accounts. By removing this it returns `true,nil` below for all secrets.